src/macros/count.rs

```diff
@@ -1,4 +1,5 @@
 /// Make an arbitrary change to a StatsD counter
+/// NOTE: avoid high tag cardinality. See note in lib docs!
```

Comment:
I'd like to be a bit more specific in these notes, so that the user knows if they are doing something wrong without having to jump to the top level docs. (feel free to rephrase it)
```suggestion
/// NOTE: do not pass values that vary significantly into tags (like IDs and timestamps) to avoid high tag cardinality. See note in lib docs!
```

Reply:
Done